scripts/ci_install_dependency.sh

```diff
@@ -19,3 +19,6 @@ pip install cutex
 
 # For compling xgrammar kernels
 pip install cuda-python nvidia-cuda-nvrtc-cu12
+
+# For router
+pip install sglang-router
```

Comment:
don't add this for now. the router is not well supported on all linux distribution yet. i have a separate ci test for router.

Reply:
Well. So how can we run the router in the docs? Can we make it pure markdown without running codes? @merrymercy